packages/labs/scoped-registry-mixin/README.md

```diff
@@ -0,0 +1,43 @@
+# @lit-labs/scoped-registry-mixin
+
+`UseScopedRegistry` mixin for `LitElement` that integrates with the speculative [Scoped CustomElementRegistry polyfill](https://github.com/webcomponents/polyfills/tree/master/packages/scoped-custom-element-registry).
+
+🚨 **Warning: Scoped Custom Element Registries is a speculative browser API that has not shipped and may change.** This mixin is provided to evaluate the spec proposal and provide feedback. **Use this feature and the polyfill in production code at your own risk.**
+
+## Overview
+
+The [Scoped Custom Element Registries](https://github.com/WICG/webcomponents/blob/gh-pages/proposals/Scoped-Custom-Element-Registries.md) WICG proposal introduces new APIs for scoping custom elements to ShadowRoots, such that the tagName-to-custom element class mapping is local per Shadow Root.
```

Comment:
```suggestion
The [Scoped Custom Element Registries](https://github.com/WICG/webcomponents/blob/gh-pages/proposals/Scoped-Custom-Element-Registries.md) WICG proposal introduces new APIs for scoping custom element definitions to shadow roots, such that the mapping of tag names to custom element class doesn't rely on a single global registry.

When new elements are created within a shadow root with a scoped custom element registry, the browser will use the scoped registry rather than the global registry to look up custom element definitions. When using a scoped element registry, all used custom element elements must be defined within that scope.

`UseScopedRegistry` adds the following features to LitElement:
1. Automatically creates a scoped CustomElementRegistry for the class
2. Provides sugar for defining custom elements in the scoped registry
3. Passes the registry to `attachShadow()` to enable scoping
4. Passes the shadow root to lit-html to create DOM within the scope
```

Reply:
done